modules/apps/frontend-js/test.properties

```diff
@@ -14,9 +14,14 @@
     #
 
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
-        (environment.acceptance == true) AND \
-        (portal.acceptance != quarantine) AND \
-        (portal.upstream != quarantine) AND \
+        (
+            (environment.acceptance == true) OR \
+            (testray.component.names ~ "AUI")
+        ) AND \
+        (
```

Comment:
@john-co , I believe the syntax is wrong. For lines, 17, 20, 21, there should be a `\` at the end of the line. Could you try that and run again?

Reply:
Ah, thanks @kenjiheigel!